godump_test.go

```diff
@@ -749,21 +742,6 @@ func TestPrintDumpHeader_SkipWhenNoFrame(t *testing.T) {
 	assert.Equal(t, "", b.String()) // nothing should be written
 }
 
-var runtimeCaller = runtime.Caller
-
-func TestCallerLocation_Fallback(t *testing.T) {
-	// Override runtime.Caller behavior
-	orig := runtimeCaller
-	defer func() { runtimeCaller = orig }()
-	runtimeCaller = func(skip int) (uintptr, string, int, bool) {
-		return 0, "", 0, false
-	}
-
-	file, line := callerLocation(5)
-	assert.Equal(t, "", file)
-	assert.Equal(t, 0, line)
-}
```

Comment:
I didn't [suggest](https://github.com/goforj/godump/pull/31#discussion_r2181487886) you to remove callerLocation as I already had a PR for it

- #28

But why not